Sources/ReplyService/FacebookGraph.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  FacebookGraph.swift
+//  SwiftBot
+//
+//  Created by Andrew Tokarev on 27/06/2017.
+//
+//
+
+import Foundation
+import PerfectCURL
+
+enum FacebookGraphRequestHttpMethod: Hashable, CustomStringConvertible {
+    
+    case get
+    case post
+    
+    public var description: String {
+        switch self {
+        case .get:      return "GET"
+        case .post:     return "POST"
+        }
+    }
+}
+
+struct FacebookGraphRequest {
+    
+    let path: String                // /me/messages
+    let params: [String: String]    // access_token:xxx
```

Comment:
If we decide to use the same classes for different GraphQL requests than they might have other parameters and don't have access token.

Idea was to create it as generic as possible and use each option as necessary